bindings/python/bt2/bt2/fields.py

```diff
@@ -366,13 +366,9 @@ def integer_field(self):
     def _set_value(self, value):
         self.integer_field.value = value
 
-    def __str__(self):
-        mapping_name = None
-        for mapping in self.mappings:
-            if self._value >= mapping.lower and self._value <= mapping.upper:
-                mapping_name = mapping.name
-                break
-        return '{}({})'.format(mapping_name, self._value)
+    def __repr__(self):
+        return '{} ({})'.format(self._value, ', '.join(
```

Comment:
Don't hesitate to use variables to avoid weird line breaks here:

    labels = [repr(v.name) for v in self.mappings]
    return '{} ({})'.format(self._value, ', '.join(labels))

Reply:
Oh, so that's what those are for?

Reply:
y para